src/lib/token_list_gen.ts

```diff
@@ -244,15 +250,15 @@ export const generateTokenList = async (
       const symbol = sanitizeSymbolString(_symbol);
 
       const arbTokenInfo: ArbTokenInfo = {
-        chainId: +l2.network.chainID,
+        chainId: +l2.network.chainId,
```

Comment:
i know this is legacy code, but we should change + to `Numbe()`, it is quite an ugly way to cast

Reply:
Agree